frontend/src/component/project/NewProjectCard/ProjectOwners/ProjectOwners.tsx

```diff
@@ -0,0 +1,75 @@
+import type { FC } from 'react';
+import { styled } from '@mui/material';
+import { GroupCardAvatars } from 'component/admin/groups/GroupsList/GroupCard/GroupCardAvatars/NewGroupCardAvatars';
+import type { ProjectSchema, ProjectSchemaOwners } from 'openapi';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+interface IProjectOwnersProps {
+    owners?: ProjectSchema['owners'];
+}
+
+const useOwnersMap = () => {
+    const { uiConfig } = useUiConfig();
+
+    return (
+        owner: ProjectSchemaOwners[0],
+    ): {
+        name: string;
+        imageUrl?: string;
+        email?: string;
+        description?: string;
+    } => {
+        if (owner.ownerType === 'user') {
+            return {
+                name: owner.name,
+                imageUrl: owner.imageUrl || undefined,
+                email: owner.email || undefined,
+            };
+        }
+        if (owner.ownerType === 'group') {
+            return {
+                name: owner.name || '',
```

Comment:
How can name be missing here? It's always there, right? Otherwise, there is something wrong with the data and it's not worth showing?